# max rate of the planner
RATE: 25
# goal tolerance
XY_GOAL_TOLERANCE: 0.02
YAW_GOAL_TOLERANCE: 0.1
# maximum speed
V_MAX: 0.35
W_MAX: 1.5
# acceleration at beginning of movements
ACCELERATION: 1.0
# goal distance for deceleration
D_DECELERATION: 0.15
# power of deceleration factor
GAMMA: 0.75
# params for accurate movements to water towers or cube heaps
XY_ACCURATE_GOAL_TOLERANCE: 0.005
YAW_ACCURATE_GOAL_TOLERANCE: 0.05
D_ACCURATE_DECELERATION: 0.20
NUM_RANGEFINDERS: 16
COLLISION_STOP_DISTANCE: 150
COLLISION_STOP_NEIGHBOUR_DISTANCE: 50
COLLISION_GAMMA: 2
# params for RRT*
MAX_RRT_ITERS: 1000
STEP: .4
GAMMA_RRT: 50
EPSILON_GOAL: .02
PATH_WIDTH: .5
